extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergParquetWriteInstructions.java

```diff
@@ -72,6 +72,14 @@ public int targetPageSize() {
         return ParquetInstructions.DEFAULT_TARGET_PAGE_SIZE;
     }
 
+    /**
+     * A one-to-one {@link Map map} from Deephaven to Parquet column names to use when writing deephaven tables to
+     * Iceberg tables.
+     */
+    public abstract Map<String, String> dhToParquetColumnRenames();
```

Comment:
Do we think there is value in letting user choose parquet column names for writing purposes?

Reply:
Now that you've mentioned, does seem like overkill to me. Will remove it.